source/config/localhost_config.json

```diff
@@ -7,5 +7,10 @@
        "server_cert": "",
        "server_key": "",
        "root_cert": ""
+    },
+    "cpu_affinity_configuration": {
+       "sideband_read_write": -1,
+       "stream_write": -1,
```

Comment:
This wording might be confusing because we will likely use grpc streaming in other contexts. We could have a normal "streaming read" or "streaming write" API that just calls the driver in a loop rather than using the special "moniker" system.

Also, isn't the stream-based implementation also read/write?

I think the key term here is `moniker`. Monikers are a special system for streaming that has a specific dedicated processing thread. It's less important for `sideband` because `sideband` is a moniker-specific concept. So this could be `moniker_stream_processor`, `moniker_stream`, `moniker_stream_read_write`.

Reply:
Renamed as suggested. 
kept the first one as `moniker_sideband_stream_read_write` to separate it from the newly added configuration for `moniker_stream_read_write`.